MVCapplication/src/main/java/com/softserve/if072/mvcapp/controller/StorePageController.java

```diff
@@ -33,6 +33,7 @@
 
     public static final Logger LOGGER = LogManager.getLogger(StorePageController.class);
 
+    private int idStore;
```

Comment:
if you have this as class field, then this variable is not thread safe.

For what reason did you do in such way?